### PR TITLE
[JBTM-2648] Add 500ms delay after starting the client in CrashRecovery12_Test03

### PR DESCRIPTION
**Pull request by gytis**
https://issues.jboss.org/browse/JBTM-2648

!MAIN !BLACKTIE !XTS !PERF

### CODE REVIEW

**Comment by mmusgrov** [comment]
The associated JIRA does not give any analysis of why the fix is required or why it fixes the issue.

**Comment by gytis** [comment]
Click through to the cloned issue on EAP

**Comment by mmusgrov** [comment]
OK I clicked through. The same symptom should have been effecting many tests so I guess it is a spurious failure. If your fix resolves Ondrejs test runs then I agree we should localise the fix to the one test.

**Comment by gytis** [comment]
I agree that it should technically fail in other cases, but that's the only
one failing AFAIK. Until other failures appear, I've decided to put the
workaround only in this specific test case.

File: qa/tests/src/org/jboss/jbossts/qa/junit/testgroup/TestGroup_crashrecovery12.java [hunk 1]
```diff
@@ -124,4 +125,12 @@ public String getTestGroupName()
 		outcome0.start("CR12_07.log", "no");
 		outcome0.waitFor();
 	}
+
+	private void suspend(long millis) {
+		try {
+			Thread.sleep(millis);
+		} catch (InterruptedException e) {
+			System.out.println("Suspend interrupted: " + e.getMessage());
```
Comment (mmusgrov) [inline qa/tests/src/org/jboss/jbossts/qa/junit/testgroup/TestGroup_crashrecovery12.java]: Normally there is a reason the thread has been interrupted so you should either re-throw the  InterruptedException or re-enable the interrupt status with:

`Thread.currentThread().interrupt() `

Comment (mmusgrov) [inline qa/tests/src/org/jboss/jbossts/qa/junit/testgroup/TestGroup_crashrecovery12.java]: There is good write up of best practice here: http://www.ibm.com/developerworks/java/library/j-jtp05236/index.html

**Comment by gytis** [comment]
@mmusgrov I've added Thread.currentThread().interrupt()

**Pull request merged**